apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -52,32 +52,81 @@ When considering compute upgrades, assess whether your bottlenecks are hardware-
 
-### Bursting and disk IO budget
+Performance Limits:
 
-Smaller compute instances can burst up to 1,048 Mbps and 3,000 IOPS for 30 minutes in a day. Beyond that, the performance reverts to the baseline. For example, the micro plan can burst up to 1,048 Mbps for 30 minutes a day and reverts to the baseline performance of 87 Mbps. Your disk IO budget gets replenished throughout the day.
+Disk throughput is capped at 19,000 Mbps and IOPS is capped at 80,000 IOPS for all compute add-ons.
 
-If you need consistent disk performance, choose the XL or larger compute add-on which has the same baseline and maximum disk throughput and IOPS.
+| Plan        | Max Disk Throughput | Baseline Disk Throughput | Max IOPS    | Baseline IOPS |
+| ----------- | ------------------- | ------------------------ | ----------- | ------------- |
+| Nano (free) | 2,085 Mbps          | 43 Mbps                  | 11,800 IOPS | 250 IOPS      |
+| Micro       | 2,085 Mbps          | 87 Mbps                  | 11,800 IOPS | 500 IOPS      |
+| Small       | 2,085 Mbps          | 174 Mbps                 | 11,800 IOPS | 1,000 IOPS    |
+| Medium      | 2,085 Mbps          | 347 Mbps                 | 11,800 IOPS | 2,000 IOPS    |
+| Large       | 4,750 Mbps          | 630 Mbps                 | 20,000 IOPS | 3,600 IOPS    |
+| XL          | 4,750 Mbps          | 1,188 Mbps               | 20,000 IOPS | 6,000 IOPS    |
+| 2XL         | 4,750 Mbps          | 2,375 Mbps               | 20,000 IOPS | 12,000 IOPS   |
+| 4XL         | 4,750 Mbps          | 4,750 Mbps               | 20,000 IOPS | 20,000 IOPS   |
+| 8XL         | 9,500 Mbps          | 9,500 Mbps               | 40,000 IOPS | 40,000 IOPS   |
+| 12XL        | 14,250 Mbps         | 14,250 Mbps              | 50,000 IOPS | 50,000 IOPS   |
+| 16XL        | 19,000 Mbps         | 19,000 Mbps              | 80,000 IOPS | 80,000 IOPS   |
+
+### Self-Serve Configurability
+
+You can now adjust storage type, IOPS and disk size parameters yourself through the Dashboard [Database Settings page](https://supabase.com/dashboard/project/_/settings/database).
+
+![Disk management configuration](/docs/img/guides/platform/disk-mgmt-io.png)
+
+<Admonition type="caution">
+
+Increasing IOPS or throughput incurs additional charges.
+
+</Admonition>
+
+### Understanding I/O Balance and Bursting (Smaller Instances)
+
+For smaller instances like Nano, Micro, and Small, you get I/O balance and burst capacity. These instances have limited base performance for IOPS and throughput. However, they come with burst credits that let you handle temporary spikes for 30 minutes in a day on demand by briefly exceeding their baseline performance. This means you can achieve higher performance for short bursts before reverting to your baseline limits.
+
+For example, the micro plan can burst up to 2,085 Mbps for 30 minutes a day and reverts to the baseline performance of 87 Mbps. Your disk IO budget gets replenished throughout the day.
```

Comment:
micro instance

We're using instance/add-on/plan interchangeably in this doc. Would be best to standardize on one - I think add-on is the one that corresponds with our Pricing structure